### ISSUE TITLE
(MV) Tests for StorageObjectVersion#verify_version_storage

### ISSUE THREAD

**Issue by SaravShah** [sul-dlss/moab-versioning#128]
- Current tests only test for the expected case, not for when they fail, or false. 
- Need to add similar tests to `#verify_manifest_inventory`, `#verify_signature_catalog`, and `#verify_version_inventory`.

**Comment by atz** [comment]
I agree, many of the tests for `Moab::StorageObjectValidator` include bogus operations that do nothing and expectations that don't provide any specificity.  This looks like it needs work.

**Comment by atz** [comment]
Also, there was a `Lint/UselessAssignment` exclusion put in `.rubocop.yml` to ignore the meaningless assignments in `spec/unit_tests/moab/storage_object_validator_spec.rb` with the comment:
```
NOT a useless assignment - rubocop is confused
```
Uh.... no, those **are** useless. Author was confused.
